tests/view/observer/fakeselectionobserver.js

```diff
@@ -100,6 +100,60 @@ describe( 'FakeSelectionObserver', () => {
 		);
 	} );
 
+	it( 'should fire `selectionChangeDone` event after selection stop changing', ( done ) => {
+		// Note that it's difficult to test lodash#debounce with sinon fake timers.
+		// See: https://github.com/lodash/lodash/issues/304
+
+		const spy = sinon.spy();
+
+		viewDocument.on( 'selectionChangeDone', spy );
+
+		// Change selection.
+		changeFakeSelectionPressing( keyCodes.arrowdown );
+
+		// Wait 100ms.
+		setTimeout( () => {
```

Comment:
Looks like there is a way to override lodash internal `Date.now` and `setTimeout` but to do it we have to use lodash `_` object instead of single import.